webgpu.h

```diff
@@ -1398,11 +1398,35 @@ typedef struct WGPUAdapterInfo {
 
 typedef struct WGPUBindGroupEntry {
     WGPUChainedStruct const * nextInChain;
+    /**
+     * Binding index in the bind group.
+     */
     uint32_t binding;
+    /**
+     * Set this if the binding is a buffer object.
+     * Otherwise must be null.
+     */
     WGPU_NULLABLE WGPUBuffer buffer;
+    /**
+     * If the binding is a buffer, this is the byte offset of the binding range.
+     * Otherwise ignored.
+     */
```

Comment:
We still might move these into either a substruct or rename them right?

Reply:
Nvm, I just got to the update in the other issue where you mentioned we probably won't change it. The documentation should be fine then!